src/migrations/default-strategies.json

```diff
@@ -4,18 +4,6 @@
     "description": "Default on/off strategy.",
     "parameters": []
   },
-  {
-    "name": "userWithId",
-    "description": "Active for users with a userId defined in the userIds-list",
-    "parameters": [
-      {
-        "name": "userIds",
-        "type": "list",
-        "description": "",
-        "required": false
-      }
-    ]
-  },
```

Comment:
This affects an existing migration, `20170211090541-add-default-strategies.js`, so same as /Users/nunogois/dev/unleash/unleash/src/migrations/20170211090541-add-default-strategies.js